src/program/lwaftr/doc/CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 # Change Log
 
+## [2018.03.01]
+
+* Features:
+
+   - Added limit-finding loadtester.
+   - Move "loadtest" command out of lwaftr.
+   - Process tree runs data-plane processes with busywait=true by default
+   - Sizes for "packetblaster lwaftr" are frame sizes
+
+* Bug fixes:
+
+   - Fix next-hop discovery with multiple devices.
+   - Remove early lwAFTR NUMA affinity check
+   - Make minimum ctable size uniform
```

Comment:
Release notess should show user-visible changes and I don't think this last one is user-visible, afaiu.  For the others, can you link to the relevant issues?